src/app/pages/menu/users/user-details/user-details.page.html

```diff
@@ -32,7 +32,7 @@
 </ion-footer>
 
 <ion-footer *ngIf="user.id">
-  <ion-row no-padding text-center>
+  <ion-row class="ion-no-padding text-center">
```

Comment:
Maybe `ion-text-center`? I dunno...

Reply:
For the sake of consistency, yes. But I didn't see a warning in the console.